solidity/contracts/middleware/InterchainQueryRouter.sol

```diff
@@ -69,11 +69,12 @@ contract InterchainQueryRouter is Router {
         address _to,
         bytes memory _data,
         bytes memory _callback
-    ) public returns (bytes32 messageId) {
+    ) public payable returns (bytes32 messageId) {
```

Comment:
note these functions actually didnt accept value before but because msg.value wasnt inline the compiler didnt catch this!